xds/internal/balancer/outlierdetection/balancer.go

```diff
@@ -829,7 +829,7 @@ func (b *outlierDetectionBalancer) successRateAlgorithm() {
 		requiredSuccessRate := mean - stddev*(float64(ejectionCfg.StdevFactor)/1000)
 		if successRate < requiredSuccessRate {
 			channelz.Infof(logger, b.channelzParentID, "SuccessRate algorithm detected outlier: %s. Parameters: successRate=%f, mean=%f, stddev=%f, requiredSuccessRate=%f",
-				addrInfo.string(), successRate, mean, stddev, requiredSuccessRate)
+				addrInfo, successRate, mean, stddev, requiredSuccessRate)
```

Comment:
I think it would be better to have this on a single line. See: https://g3doc.corp.google.com/go/g3doc/style/decisions.md?cl=head#indentation-confusion. Specifically: https://screenshot.googleplex.com/4R2ZBD6WsCKv6kj.

Reply:
Also, do we have an idea how this line appears in the log? Like, I don't know if the `addrInfo` is readable in the logs.

Reply:
Ah, this was leftovers from the fact he did it in Java first. Switched.

Reply:
Logs look like 
`
FailurePercentage algorithm detected outlier: [[{
          "Addr": "localhost:60860",
          "ServerName": "",
          "Attributes": {},
          "BalancerAttributes": {},
          "Type": 0,
          "Metadata": null
        }]], failurePercentage=100.000000
`
This looks good, and as you will see in my upcoming wrr_locality PR I changed this String() method on the address to be much more readable all the way down to the Balancer Attributes/Attributes layer.

Reply:
Nice.